pymc3/distributions/distribution.py

```diff
@@ -328,6 +337,24 @@ def dist(
         return rv_out
 
 
+@singledispatch
+def _get_moment(op, rv, size, *rv_inputs) -> TensorVariable:
+    """Fallback method for creating an initial value for a random variable.
+
+    Parameters are the same as for the `.dist()` method.
+    """
+    return None
+
+
+def get_moment(rv: TensorVariable) -> TensorVariable:
+    """Fallback method for creating an initial value for a random variable.
+
+    Parameters are the same as for the `.dist()` method.
```

Comment:
I know this was merged already, but this part of the docstrings is wrong

Reply:
OK, we should fix that then. CC @kc611

Reply:
Ah yes I missed that, those docstrings were supposed to be removed. 

I'm not sure what (docstring) will go in it's place though. Should I just remove them for now ? Maybe we can add a proper explanation when we give the `get_moment` a proper entry point in the `initval` framework (if that's being planned)

Reply:
Yes, then just remove them for now.

Reply:
Did it in https://github.com/pymc-devs/pymc3/pull/4979